### PR TITLE
erigon: 2021.09.02 -> 2021.09.04

### PR DESCRIPTION
**Pull request by iceman-p**
Semi-automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. 

meta.description for erigon is: "Ethereum node implementation focused on scalability and modularity"

meta.homepage for erigon is: "https://github.com/ledgerwatch/erigon/"

meta.changelog for erigon is: ""

###### Updates performed
- Golang update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/ledgerwatch/erigon/releases/tag/v2021.09.04)

- [Compare changes on GitHub](https://github.com/ledgerwatch/erigon/compare/v2021.09.02...v2021.09.04)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04/bin/rlpdump passed the binary check.
- /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04/bin/evm passed the binary check.
- /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04/bin/erigon passed the binary check.
- /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04/bin/rpcdaemon passed the binary check.
- 4 of 4 passed binary check by having a zero exit code.
- 1 of 4 passed binary check by having the new version present in output.
- found 2021.09.04 in filename of file in /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1 total rebuild path(s)

1 package rebuild(s)

0 x86_64-linux rebuild(s)
0 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
0 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
erigon
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Build yourself:
```
nix-build -A erigon https://github.com/iceman-p/nixpkgs/archive/30bd573c0cb047458373c8dd3c15c6d4a0bad1ca.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04
ls -la /nix/store/0cq5mjpq76nxi3ylpybdcs8i5ki9x1j5-erigon-2021.09.04/bin
```

---

</details>
<br/>

### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>erigon</li>
  </ul>
</details>

---

###### Maintainer pings

cc @d-xo @kalbasit for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

### CODE REVIEW

**Comment by bobby285271** [comment]
@ofborg eval

File: pkgs/applications/blockchains/erigon.nix [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
   pname = "erigon";
-  version = "2021.09.02";
+  version = "2021.09.04";
 
   src = fetchFromGitHub {
     owner = "ledgerwatch";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-0rWyDlZjfsZMOqAXs+mgmgz0m4oIN6bZ6Z9U4jWgR0E=";
+    sha256 = "sha256:0l0w1badhvlh1rgqzvlmy5k7xhb1nf4f5dmhkl935a5ila08aak3";
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/erigon.nix:11]:
```suggestion
    sha256 = "0l0w1badhvlh1rgqzvlmy5k7xhb1nf4f5dmhkl935a5ila08aak3";
```

File: pkgs/applications/blockchains/erigon.nix [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
   };
 
-  vendorSha256 = "sha256-ardr+6Tz9IzSJPo9/kk7XV+2pIu6ZK3YYlp1zC/7Bno=";
+  vendorSha256 = "sha256:1hbfmq76zm50zwmlh3jblriwq2k1mp99d8lg8xzxwy56hncgfj8k";
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/erigon.nix:14]:
```suggestion
  vendorSha256 = "1hbfmq76zm50zwmlh3jblriwq2k1mp99d8lg8xzxwy56hncgfj8k";
```

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**